Gordon360/Services/HousingService.cs

```diff
@@ -737,5 +781,22 @@ public ApartmentApplicationViewModel[] GetAllApartmentApplication()
             }
             return apartmentApplicationArray;
         }
+
+        // I appreciate that we are not typing redundant comments,
+        // but can we have something to make it more clear where these services start?
```

Comment:
These comments seem like they are intended as a question for your team, and not as explanation/documentation of the code.